### PR TITLE
Add video view to allow video preview of games in gamelist

### PR DESCRIPTION
**Pull request by fieldofcows**
This feature has been discussed in https://retropie.org.uk/forum/topic/4820/video-preview-in-emulationstation and a request for testing has been submitted in https://retropie.org.uk/forum/topic/5823/looking-for-testers-for-es-video-preview-on-raspberry-pi. Please refer to these threads for details on the implementation, theming and testing.

### CODE REVIEW

**Comment by joolswills** [comment]
Many thanks for your work on this. Have been following the discussion - looks good. Will give it a test and get back to you/merge. (I will need to update the RetroPie-Setup script to include the additional dependencies also).

File: README.md [hunk 1]
```diff
@@ -41,7 +41,10 @@ EmulationStation has a few dependencies. For building, you'll need CMake, SDL2,
 **On Debian/Ubuntu:**
 All of this be easily installed with apt-get:
 ```bash
-sudo apt-get install libsdl2-dev libboost-system-dev libboost-filesystem-dev libboost-date-time-dev libboost-locale-dev libfreeimage-dev libfreetype6-dev libeigen3-dev libcurl4-openssl-dev libasound2-dev libgl1-mesa-dev build-essential cmake fonts-droid
+sudo apt-get install libsdl2-dev libboost-system-dev libboost-filesystem-dev libboost-date-time-dev \
+  libboost-locale-dev libfreeimage-dev libfreetype6-dev libeigen3-dev libcurl4-openssl-dev \
+  libasound2-dev libgl1-mesa-dev build-essential cmake fonts-droid \
+  libvlc5 libvlc-dev libvlccore8 libvlccore-dev
```
Comment (joolswills) [inline README.md]: After installing `vlc-plugin-sdl` video plays fine on the compiled ES.

Please can you update the dependencies / squash and I will merge. thanks.

Comment (joolswills) [inline README.md]: I assume why `vlc-plugin-sdl` was needed was due to the library files it depended on being needed (libavcodec etc). vlc-plugin-sdl isn't needed directly or ?

**Comment by fieldofcows** [comment]
I've now made the requested changes to README.md (remove non-dev packages and add missing dependency) and I've also added a fix to the video component that can result in the video being displayed with the wrong opacity in some themes.

These two changes have been squashed into the original commit.

**Comment by joolswills** [comment]
Great - thanks again for this excellent enhancement.

**Pull request merged**